fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -863,6 +864,7 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 		std::shared_ptr<ReadIteratorPool> readIterPool;
 		std::shared_ptr<PerfContextMetrics> perfContextMetrics;
 		int threadIndex;
+		ThreadReturnPromiseStream<std::tuple<int, const StringRef, double>>* metricPromiseStream;
```

Comment:
We could store ThreadReturnPromiseStream directly, so we don't need to explicitly create the object in line 877.